src/utils/api.js

```diff
@@ -1,7 +1,7 @@
 const $ = require('jquery');
 const querystring = require('querystring');
 
-const API_ENDPOINT = 'https://api.betterttv.net/2/';
+const API_ENDPOINT = 'https://api.betterttv.net/3/cached/';
```

Comment:
should leave off `cached` bit, since that is part of the path .. we might hit stuff not being cached

Reply:
Should the cache path only be used for emotes i assume? accessing https://api.betterttv.net/3/users/twitch/107780543 returns 404 without the /cached part